libs/shared/ui/phone-number.tsx

```
@@ -96,7 +96,7 @@ function CountrySelect({
           disabled={disabled}
         >
 
-          <img src={chevronUpDown} className="h-6 w-6" />
+          <img src={chevronUpDown} className="size-6" />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)